### PR TITLE
HOT FIX: Fix bug where unexpected error shown along with normal error message

### PR DESCRIPTION
**Pull request by OmegaOoh**
### Description

- Fix bug where unexpected error shown along with normal error message

### Functional Acceptance Criteria
Everything should works the same and no more double alert message

### Checklist

- [x] I have performed a self-review of my code.
- [ ] I have added tests to cover my changes.
- [ ] I have updated the documentation, if applicable.
- [x] My code follows the Coding Guidelines.

### Type of Change

- [x] Bug fix
- [ ] New feature
- [ ] Documentation update
- [ ] Other (please specify)

### CODE REVIEW

File: frontend/src/component/CheckInModal.vue [hunk 1]
```diff
@@ -86,36 +86,16 @@ const checkInCode = ref('');
...
-                );
+        const response = await createPostRequest(
+            `/activities/check-in/${props.id}/`,
+            {
+                check_in_code: checkInCode.value,
             }
-        }
+        );
+        if (!response) return; // Failed
+        addAlert('success', response.data.message);
```
Comment (opxz7148) [inline frontend/src/component/CheckInModal.vue:96]: I'm not totally understand this change. What if there are error occur from the backend will the message show as green?

Comment (OmegaOoh) [reply frontend/src/component/CheckInModal.vue:96]: Check HttpRequest.js It does handle the error and return none. Then if there is error occurs this code fragments need to returned

Comment (opxz7148) [reply frontend/src/component/CheckInModal.vue:96]: Ok I forget that it's not bare apiClient then it's reasonable.

**Review by opxz7148 — APPROVED** [review]
Other then fixing the bug this make code way more clean, nice _Approve_

**Review by Thanawas-Sirilertsathit — APPROVED** [review]
Approved

**Pull request merged**